package.json

```diff
@@ -126,6 +126,7 @@
   "devDependencies": {
     "@babel/core": "7.9.0",
     "@babel/polyfill": "7.8.7",
+    "@babel/plugin-transform-runtime": "^7.10.0",
```

Comment:
Oh, you also forgot to not include the caret.
This way we also lock the used version in our package.json